on-boarding/codereview.dart

```diff
@@ -0,0 +1,187 @@
+import 'dart:io';
+
+class Task {
+  String? title;
```

Comment:
try making the title a required value rather that making it nullable, use the "required" key work in the constructor
Same for the description as well

Reply:
resolved